cypress/integration/cucumber/features/login.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Login
+
```

Comment:
You need to add tags to your scenario

Reply:
Let's first work on getting Cucumber working in this scenario, get some tests built and then we can start diving things using tags. For now, with only a single feature file, they don't really add any value.

Reply:
> get some tests built and then we can start diving things using tags

I agree!